interop-cats/shared/src/test/scala/zio/interop/catzSpecBase.scala

```diff
@@ -50,4 +51,24 @@ private[interop] sealed trait catzSpecBaseLowPriority { this: catzSpecBase =>
     Eq.instance((io1, io2) => Arbitrary.arbitrary[R].sample.fold(false)(r => catsSyntaxEq(run(r, io1)) eqv run(r, io2)))
   }
 
+  //---!! TODO: this is the comment for a reviewer: remove or rewrite it after the review!!---
```

Comment:
We've got to figure out a way to organize the instances so we don't have these issues? Maybe we can mirror the organization structure for the `ZIO` instances, which seem to be working for instances including ones with an environment type.

Reply:
Fixed. Thanks for your advise.
The problem is really strange because previously Eq for ZManaged[Any, E, A] was even in higher priority scope then now